joomla/joomla-cms
# [4.0] Move sidebar modules into grid

Pull Request for Issue # .

### Summary of Changes
Moves the sidebar modules into the main CSS grid by using `has-*` classes.

- Less nested divs
- Less hacky
- Potential scope for IE11 CSS grid support (using old grid spec). 

For a more detailed explanation.. https://joomla51.com/blog/item/joomla-css-grid-and-dynamic-content

### Testing Instructions
Apply patch (cant use patchtester) and compile CSS. Check frontend layout with modules enabled/disabled in the **sidebar-left** and **sidebar-right** modules positions
